apps/settings/locales/settings.en-US.properties

```diff
@@ -1233,7 +1233,8 @@ hud-security=Security Issues
 hud-performance=Performance
 hud-reflows=Reflows
 hud-jank=Jank
-jank-threshold=Jank Threshold
+jank-threshold=Jank Threshold (ms)
```

Comment:
please rename the id when you change the string, or the l10n team can't fetch and localize it